spec/features/events_spec.rb

```diff
@@ -22,6 +22,45 @@
     expect(page).to have_css("div#occupied_places")
   end
 
+  scenario "logged in as Organizer I want to be unable to send emails if there is any unclassified application left" do
+    login(:organizer)
+    @event.update!(max_participants: 1)
+    @pupil = FactoryGirl.create(:profile)
+    @pupil.user.role = :pupil
+    @pending_application = FactoryGirl.create(:application_letter, :event => @event, :user => @pupil.user)
+    visit event_path(@event)
+    expect(page).to have_button(I18n.t('events.applicants_overview.sending_acceptances'), disabled: true)
+    expect(page).to have_button(I18n.t('events.applicants_overview.sending_rejections'), disabled: true)
+  end
+
+  scenario "logged in as Organizer I want to be unable to send emails if there is a negative number of free places left" do
+    login(:organizer)
+    @event.update!(max_participants: 1)
+    @pupil1 = FactoryGirl.create(:profile)
+    @pupil1.user.role = :pupil
+    @pupil2 = FactoryGirl.create(:profile)
+    @pupil2.user.role = :pupil
+    @accepted_application = FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil1.user)
+    @accepted_application_2 = FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil2.user)
+    visit event_path(@event)
+    expect(page).to have_button(I18n.t('events.applicants_overview.sending_acceptances'), disabled: true)
+    expect(page).to have_button(I18n.t('events.applicants_overview.sending_rejections'), disabled: true)
+  end
+
+  scenario "logged in as Organizer I want to open a modal by clicking on sending emails" do
+    login(:organizer)
+    @event.update!(max_participants: 2)
+    @pupil1 = FactoryGirl.create(:profile)
+    @pupil1.user.role = :pupil
+    @pupil2 = FactoryGirl.create(:profile)
+    @pupil2.user.role = :pupil
+    @accepted_application = FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil1.user)
+    @accepted_application_2 = FactoryGirl.create(:application_letter_accepted, :event => @event, :user => @pupil2.user)
```

Comment:
See [previous test](https://github.com/hpi-swt2/workshop-portal/pull/107/files#diff-f20a1b372768b0b5eebe83bc6aa154f8R36) If this snippet is more frequently used it could be extracted in a separate method, but I think that is currently not essential.